R/tools.R

```diff
@@ -122,12 +137,22 @@ rhino_style <- function() {
 
 #' Format R
 #'
-#' Uses the `{styler}` package to automatically format R sources.
+#' Uses the `{styler}` and `{box.linters}` packages to automatically format R sources.
 #'
 #' The code is formatted according to the `styler::tidyverse_style` guide with one adjustment:
 #' spacing around math operators is not modified to avoid conflicts with `box::use()` statements.
 #'
+#' `box.linters::style_*` functions require the `treesitter` and `treesitter.r` packages. These, in
+#' turn, require R >= 4.3.0. `box.linters` provides styling functions specific to the `box::use()`
```

Comment:
Please change the order of those sentences. The information that `box.linters` provides styling for `box::use` should go first, followed by the R version requirement.